build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -497,3 +498,40 @@ export function filterVersionsOlderThan(
         return diff <= numBusinessDays;
     });
 }
+
+export function getFluidDependencies(
+    context: Context,
+    releaseGroupOrPackage: ReleaseGroup | ReleasePackage,
+): [releaseGroups: PackageVersionMap, packages: PackageVersionMap] {
+    const releaseGroups: PackageVersionMap = {};
+    const packages: PackageVersionMap = {};
+    let packagesToCheck: Package[];
+
+    if (isReleaseGroup(releaseGroupOrPackage)) {
+        packagesToCheck = context.packagesInReleaseGroup(releaseGroupOrPackage);
+    } else {
+        const independentPackage = context.fullPackageMap.get(releaseGroupOrPackage);
+        assert(
+            independentPackage !== undefined,
+            `Package not found in context: ${releaseGroupOrPackage}`,
+        );
+        packagesToCheck = [independentPackage];
+    }
+
+    for (const p of packagesToCheck) {
+        for (const dep of p.combinedDependencies) {
+            const pkg = context.fullPackageMap.get(dep.name);
+            if (pkg !== undefined) {
+                if (pkg.monoRepo === undefined) {
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    packages[pkg.name] = semver.minVersion(dep.version)!.version;
+                } else {
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    releaseGroups[pkg.monoRepo.kind] = semver.minVersion(dep.version)!.version;
+                }
+            }
```

Comment:
considering https://typescript-eslint.io/rules/strict-boolean-expressions that would become

```suggestion
            if (pkg === undefined) continue;
            if (pkg.monoRepo !== undefined) {
                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                releaseGroups[pkg.monoRepo.kind] = semver.minVersion(dep.version)!.version;
                continue;
            }
```

Reply:
Fixed in latest iteration.